src/bot/scripts/card.js

```diff
@@ -967,96 +1016,97 @@ const cb = async (client, message) => {
     embed.setDescription("");
 
     embed.description += "**General Information**\n";
-    if (cardData.cast_area === "own_side") {
+    if (card.cast_area === "own_side") {
       embed.description += "Cast Area: Own Side\n";
     } else {
-      embed.description += `Cast Area: ${cardData.cast_area}\n`;
+      embed.description += `Cast Area: ${card.cast_area}\n`;
     }
-    if (cardData.character_type !== "totem") {
+    if (card.character_type !== "totem") {
       embed.description += `Max Speed: ${Math.round(
-        cardData.max_velocity * 100
+        card.max_velocity * 100
       ) / 100}\n`;
       embed.description += `Time To Reach Max Speed: ${Math.round(
-        cardData.time_to_reach_max_velocity * 100
+        card.time_to_reach_max_velocity * 100
       ) / 100}\n`;
       embed.description += `Agro Range Multiplier: ${Math.round(
-        cardData.agro_range_multiplier * 100
+        card.agro_range_multiplier * 100
       ) / 100}\n\n`;
     } else {
       embed.description += "\n";
     }
 
     let powers = [];
-    if ((cardData.powers == null ? 0 : cardData.powers.length) !== 0) {
+    if ((card.powers == null ? 0 : card.powers.length) !== 0) {
       embed.description += "**Power Information? - Yes**\n";
 
-      cardData.powers.forEach(e => {
-        let power = {};
-
-        if (e.type !== null) {
+      let power = {};
+      for (let statpowers of stats.powers) {
+        if (statpowers.type !== null) {
           embed.description += `Power Type: ${removeUnderscores(
-            e.type
-          )} \nPower Amount: ${e.amount}\n`;
+            statpowers.type
+          )} \nPower Amount: ${statpowers.amount}\n`;
         }
 
-        if (e.duration != null) {
-          embed.description += `Power Duration: ${e.duration} \n`;
+        if (statpowers.duration != null) {
+          embed.description += `Power Duration: ${statpowers.duration} \n`;
         }
 
-        if (e.is_charged) {
-          embed.description += `Charged Power Regen: ${e.charged_regen}\n`;
-          embed.description += `Charged Power Radius: ${e.radius} \n`;
+        if (statpowers.is_charged && card._id !== "5caba68cc83b14195097bf4a" && card._id !== "5caba68ec83b14195097bf57") {
```

Comment:
Never NEVER hardcode ids, at the VERY LEAST for the simple fact that those arent bound to cards but database entries, if we for some reason need to reset the db, or delete and remake an entry, THEY WILL CHANGE. WHen youre fixing this, avoid hardocding for certain cards as well.